apps/www/registry/default/example/dialog-close-button.tsx

```diff
@@ -40,7 +41,8 @@ export default function DialogCloseButton() {
           </div>
           <Button type="submit" size="sm" className="px-3">
             <span className="sr-only">Copy</span>
-            <Copy className="h-4 w-4" />
+            <CopyButtonForDialog value="https://ui.shadcn.com/docs/installation" />
+            {/* <Copy className="h-4 w-4" /> */}
```

Comment:
and this line commented? Is it ok for something in particular?

Reply:
Sorry i dont mean to do that . it is totally my fault sorry for such confusing, it is now fixed